tests/installation/bootloader_s390.pm

```diff
@@ -86,7 +86,11 @@ sub prepare_parmfile {
 
     # we have to hardcode the hostname here - the true hostname would
     # create a too long parameter ;(
-    my $instsrc = get_var('INSTALL_SOURCE', 'ftp') . '://ftp:ftp@' . get_var('REPO_HOST', 'openqa') . '/';
+    my $repo_host = get_var('REPO_HOST', 'openqa');
+    if (check_var('INSTALL_SOURCE', 'ftp')) {
+      $repo_host = 'ftp:ftp@' . $repo_host;
+    }
+    my $instsrc = get_var('INSTALL_SOURCE', 'ftp') .  $repo_host . '/';
```

Comment:
my $instsrc = get_var('INSTALL_SOURCE', 'ftp') . '://' . $repo_host . '/';

Reply:
I believe the URL will be incorrect with the proposed change when I looked more closely at the code.

Reply:
Sure, it was just a proposal/idea.